.expeditor/announce-release.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -exou pipefail
+
+# Download the release-notes for our specific build
+curl -o release-notes.md "https://packages.chef.io/release-notes/${EXPEDITOR_PRODUCT_KEY}/${EXPEDITOR_VERSION}.md"
+
+topic_title="Chef InSpec $EXPEDITOR_VERSION Released!"
+topic_body=$(cat <<EOH
+Hello InSpec friends!
```

Comment:
Open to suggestions on the format/content contained in the release announcement. I just took a first stab at things.

Reply:
@kekaichinose, @mjingle How does this look to you?

Reply:
Where will this messaging show up? On the micro-site?

Reply:
Looks like just to Discourse so far. We could potentially put it on the microsite like Automate does (see: https://automate.chef.io/release-notes/), but the microsite would need modification.